11ty/definitions/fr_FR/pluriel.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Pluriel
```

Comment:
```suggestion
title: pluriel
```
In French is it necessary to capitalise words that are not proper nouns? It's inconsistent in our English versions (which requires addressing), but curious if this should be the case in French or not.

Reply:
It's not necessary, French generally follows the same rules as English in that regard 👍

Reply:
Ah, thank you! Relieved to hear this as I suggested making it lowercase for all terms 🙈

Reply:
Yeah that one is on me, I think it's just my habit of starting everything with an uppercase. Should be fixed everywhere now, sorry!